### PR TITLE
Correcting some misspelled words and removing unnecessary else statements.

### PR DESCRIPTION
**Pull request by d6o**
- Fix three misspelled words in the files.
- Remove some unnecessary else statements. 
- Substitute some `fmt.Fprintf` to `Fprint` when there is no need to use formatted strings (they are less expensive).

### CODE REVIEW

**Comment by d6o** [comment]
I signed it!

File: eval.go [hunk 1]
```diff
@@ -359,9 +359,8 @@ func exec(fr *Frame, stmt syntax.Stmt) error {
 		}
 		if cond.Truth() {
 			return execStmts(fr, stmt.True)
-		} else {
-			return execStmts(fr, stmt.False)
```
Comment (alandonovan) [inline eval.go]: Please leave as is.  The common indentation makes the symmetry of the two branches more obvious.  I don't mind indenting the error case and not the normal case, but that rule needn't apply when the two cases are equally normal.

File: cmd/skylark/skylark.go [hunk 2]
```diff
@@ -93,7 +93,7 @@ func repl() {
 	sc := bufio.NewScanner(os.Stdin)
 outer:
 	for {
-		fmt.Fprintf(os.Stderr, ">>> ")
+		fmt.Fprint(os.Stderr, ">>> ")
```
Comment (alandonovan) [inline cmd/skylark/skylark.go]: io.WriteString(...)

File: cmd/skylark/skylark.go [hunk 3]
```diff
@@ -118,7 +118,7 @@ outer:
 		var buf bytes.Buffer
 		fmt.Fprintln(&buf, line)
 		for {
-			fmt.Fprintf(os.Stderr, "... ")
+			fmt.Fprint(os.Stderr, "... ")
```
Comment (alandonovan) [inline cmd/skylark/skylark.go]: io.WriteString

File: int.go [hunk 4]
```diff
@@ -187,9 +187,8 @@ func ConvertToInt(x Value) (Int, error) {
 	case Bool:
 		if x {
 			return one, nil
-		} else {
```
Comment (alandonovan) [inline int.go]: Again, leave as is.

File: library.go [hunk 5]
```diff
@@ -498,9 +498,8 @@ func float(thread *Thread, _ *Builtin, args Tuple, kwargs []Tuple) (Value, error
 	case Bool:
 		if x {
 			return Float(1.0), nil
-		} else {
```
Comment (alandonovan) [inline library.go]: ditto

File: syntax/scan.go [hunk 6]
```diff
@@ -441,10 +441,9 @@ start:
 		if sc.dents < 0 {
 			sc.dents++
 			return OUTDENT
-		} else {
```
Comment (alandonovan) [inline syntax/scan.go]: ditto

File: syntax/scan.go [hunk 7]
```diff
@@ -633,9 +632,8 @@ start:
 				if sc.peekRune() == '=' {
 					sc.readRune()
 					return SLASHSLASH_EQ
-				} else {
```
Comment (alandonovan) [inline syntax/scan.go]: ditto

File: syntax/scan.go [hunk 8]
```diff
@@ -841,19 +839,18 @@ func (sc *scanner) scanNumber(val *tokenValue, c rune) Token {
 			sc.error(sc.pos, "invalid float literal")
 		}
 		return FLOAT
+	}
+	var err error
+	s := val.raw
+	if len(s) > 2 && s[0] == '0' && (s[1] == 'o' || s[1] == 'O') {
+		val.int, err = strconv.ParseInt(s[2:], 8, 64)
 	} else {
-		var err error
```
Comment (alandonovan) [inline syntax/scan.go]: ditto

File: value.go [hunk 9]
```diff
@@ -296,9 +296,8 @@ const (
 func (b Bool) String() string {
 	if b {
 		return "True"
-	} else {
```
Comment (alandonovan) [inline value.go]: ditto

File: value.go [hunk 10]
```diff
@@ -407,16 +406,15 @@ var _ Iterable = (*stringIterable)(nil)
 func (si stringIterable) String() string {
 	if si.split {
 		return si.s.String() + ".split_" + si.Type() + "()"
-	} else {
```
Comment (alandonovan) [inline value.go]: ditto

File: value.go [hunk 11]
```diff
@@ -407,16 +406,15 @@ var _ Iterable = (*stringIterable)(nil)
...
-		return si.s.String() + "." + si.Type() + "()"
 	}
+	return si.s.String() + "." + si.Type() + "()"
+
 }
 func (si stringIterable) Type() string {
 	if si.codepoints {
 		return "codepoints"
-	} else {
```
Comment (alandonovan) [inline value.go]: ditto

File: value.go [hunk 12]
```diff
@@ -1048,9 +1046,8 @@ func threeway(op syntax.Token, cmp int) bool {
 func b2i(b bool) int {
 	if b {
 		return 1
-	} else {
```
Comment (alandonovan) [inline value.go]: ditto

**Review by alandonovan — COMMENTED** [review]
Thanks for the patch!

Did you run the tests?  I would expect the error message typo to show up in an at least one assertion.

**Comment by d6o** [comment]
Thank you for the review @alandonovan !! I'll do the changes tonight!

**Comment by d6o** [comment]
PR updated! All tests are passing

```
go test ./...
ok  	github.com/google/skylark	0.036s
?   	github.com/google/skylark/cmd/skylark	[no test files]
?   	github.com/google/skylark/internal/chunkedfile	[no test files]
ok  	github.com/google/skylark/resolve	0.010s
?   	github.com/google/skylark/skylarkstruct	[no test files]
?   	github.com/google/skylark/skylarktest	[no test files]
ok  	github.com/google/skylark/syntax	0.012s
```

File: eval.go [hunk 13]
```diff
@@ -737,9 +737,8 @@ func eval(fr *Frame, e syntax.Expr) (Value, error) {
 		}
 		if cond.Truth() {
 			return eval(fr, e.True)
-		} else {
-			return eval(fr, e.False)
 		}
+		return eval(fr, e.False)
```
Comment (alandonovan) [inline eval.go]: Please revert this edit.

File: syntax/syntax.go [hunk 14]
```diff
@@ -378,9 +378,8 @@ type TupleExpr struct {
 func (x *TupleExpr) Span() (start, end Position) {
 	if x.Lparen.IsValid() {
 		return x.Lparen, x.Rparen
-	} else {
```
Comment (alandonovan) [inline syntax/syntax.go]: Please revert this edit too: neither value of the condition is more or less "normal" than the other.

**Comment by d6o** [comment]
Updated again!

**Review by alandonovan — APPROVED** [review]

**Pull request merged**